.changelog/unreleased/dependencies/3193-bump-ics.md

```diff
@@ -0,0 +1,3 @@
+- Bump [ICS](https://github.com/cosmos/interchain-security) to
+    [v4.3.1-lsm](https://github.com/cosmos/interchain-security/releases/tag/v4.3.1-
+  lsm) ([\#3193](https://github.com/cosmos/gaia/pull/3193))
```

Comment:
Move `-lsm` to the previous line? Not sure if this breaks the link.
The same for the `state-breaking` file.